Formula/djl-serving.rb

```diff
@@ -5,6 +5,12 @@ class DjlServing < Formula
   sha256 "f87189733657e80c5f3ad94a11eb73f3bb7a3433f2fbd5441662a94e0d887f9c"
   license "Apache-2.0"
 
+  # The official release notes would be in `deepjavalibrary/djl` repo
+  livecheck do
+    url "https://github.com/deepjavalibrary/djl.git"
```

Comment:
```suggestion
    url "https://github.com/deepjavalibrary/djl"
```

Since we're using the `GithubLatest` strategy here, using a `.git` URL may be a bit misleading to anyone who doesn't understand that the strategy will convert the URL to the `.../releases/latest` URL.